diff -Naur linux-3.11.1/arch/x86/tools/relocs.c linux-3.11.1-fixed/arch/x86/tools/relocs.c
--- linux-3.11.1-old/arch/x86/tools/relocs.c	2023-08-30 14:56:03.135053753 +0800
+++ linux-3.11.1/arch/x86/tools/relocs.c	2023-08-30 14:46:34.135378677 +0800
@@ -695,7 +695,7 @@
  *
  */
 static int per_cpu_shndx	= -1;
-Elf_Addr per_cpu_load_addr;
+static Elf_Addr per_cpu_load_addr;
 
 static void percpu_init(void)
 {
